### ISSUE TITLE
Add other networks like Hubzilla/Mastodon/Pleroma/Diaspora ...

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica-directory#25]
Would be nice if we start adding other networks to the friendica directory to have a location where you can search for new contacts.
 - Hubzilla
 - Mastodon
 - Pleroma
 - Diaspora
 - SocialHome
 - GangGo
 - Misskey
 - GNU Social
 - ...

**Comment by AndyHee** [comment]
Do these networks actually provide individual user information? In other words, is there for instance a global GangGo or Misskey directory of users?

**Comment by hoergen** [comment]
There are several directories like https://fediverse.network/ .
It was just a raw idea. I was thinking that every friendica server collect account informations about accounts and servers and if accounts are publishing their account this could maybe streamed to the directory or the directory fetches that information, when it contacts a friendica node ?

**Comment by annando** [comment]
Technically this is no problem. But collecting data from people who can't agree or disagree is problematic.

**Comment by AndyHee** [comment]
Good idea. At node level, it should be easy to get information about what servers are out (but that already exist as you've shown). At individual user level, it's much harder for legal and ethical reasons.

Everyone who is listed in _a_ global Friendica directory has consented to have their personal information published. Harvesting such information from individual nodes  --without permission-- of course doesn't work.

This is partially a problem of decentralisation. A way out from this would be contacting other projects and see if they are open for having their own global (user) directories. Once such an infrastructure is in place on their side, our global directories could obtain their information. 

In a sense, what we are looking for --to refine your idea-- is a network of global directories that can exchange user information.
